pkg/util/util.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+
+	genericapiserver "k8s.io/apiserver/pkg/server"
```

Comment:
nit: If I had a crystal ball, I think I'd make the prophecy that Stefan would want this helper to end up in this upstream package ... :^)

Reply:
How about moving that into the upstream package?